tests/scope_test_config.yaml

```diff
@@ -36,7 +36,7 @@ tests:
           severity: HIGH
           finding_type: REPORT
           source: CALLBACK
-          aggregation_key: crash_loop
+          aggregation_key: CrashLoop
```

Comment:
I think this test fails
You probably also need to change the identifier on line 24 as well